services/fxa/hubhandler.py

```diff
@@ -22,7 +23,44 @@
 from aws_xray_sdk.ext.flask.middleware import XRayMiddleware
 
 from hub.app import create_app
-from shared.log import get_logger
+
+from structlog import configure, processors, stdlib, threadlocal, get_logger
+from pythonjsonlogger import jsonlogger
+
+
+logging.config.dictConfig(
+        {
+            "version": 1,
+            "disable_existing_loggers": False,
+            "formatters": {
+                "json": {
+                    "format": "%(message)s %(lineno)d %(pathname)s",
+                    "class": "pythonjsonlogger.jsonlogger.JsonFormatter",
+                }
+            },
+            "handlers": {
+                "json": {"class": "logging.StreamHandler", "formatter": "json"}
+            },
+            "loggers": {"": {"handlers": ["json"], "level": CFG.LOG_LEVEL}},
+        }
+)
+
+configure(
+        context_class=threadlocal.wrap_dict(dict),
+        logger_factory=stdlib.LoggerFactory(),
+        wrapper_class=stdlib.BoundLogger,
+        processors=[
+            stdlib.filter_by_level,
+            stdlib.add_logger_name,
+            stdlib.add_log_level,
+            stdlib.PositionalArgumentsFormatter(),
+            processors.TimeStamper(fmt="iso"),
+            processors.StackInfoRenderer(),
+            processors.format_exc_info,
+            processors.UnicodeDecoder(),
+            stdlib.render_to_log_kwargs,
```

Comment:
Seems like a lot of our previous configuration for the logging has been dropped on the floor
https://github.com/mozilla/subhub/blob/master/src/shared/log.py#L55-L93